package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-jhipster-micronaut",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
Do not bump this yet, we will do this when we are releasing :)

Reply:
Got it, will change, ty :)